src/renderer/classes/outputManager/MemberChannel.js

```diff
@@ -11,45 +11,43 @@ class MemberChannel {
   /**
    * @param  {object} param
    * @param  {number} param.midiChannel Integer from 1 to 16
-   * @param  {function} param.nowFn Function to get current time. e.g.`performance.now`
+   * @param  {function} param.nowCb Function to get current time. e.g.`performance.now`
    */
-  constructor ({ midiChannel, nowFn }) {
-    this.nowFn = nowFn;
-    this.timeOfLastNoteOff = nowFn();
+  constructor ({ midiChannel, nowCb }) {
+    this.now = nowCb;
+    this.timeOfLastNoteOff = this.now();
+    this.timeOfLastNoteOn = this.now();
```

Comment:
I like this idea of retrieving a function to get time from outside the class since it seems extendible!

Reply:
Yes! It's easy to test in Node environment.